datafusion/ffi/src/udf/mod.rs

```diff
@@ -207,6 +208,8 @@ unsafe extern "C" fn invoke_with_args_fn_wrapper(
         arg_fields,
         number_rows,
         return_field,
+        // todo - should the config options go through serdes?
```

Comment:
I filed a ticket to track this:
- https://github.com/apache/datafusion/issues/17035

```suggestion
        // TODO: pass config options: https://github.com/apache/datafusion/issues/17035
```